core/src/server/rpc_module.rs

```diff
@@ -606,15 +607,16 @@ impl<Context: Send + Sync + 'static> RpcModule<Context> {
 
 	/// Register a new **blocking** synchronous RPC method, which computes the response with the given callback.
 	/// Unlike the regular [`register_method`](RpcModule::register_method), this method can block its thread and perform expensive computations.
-	pub fn register_blocking_method<R, F>(
+	pub fn register_blocking_method<R, E, F>(
 		&mut self,
 		method_name: &'static str,
 		callback: F,
 	) -> Result<MethodResourcesBuilder, Error>
 	where
 		Context: Send + Sync + 'static,
 		R: Serialize,
-		F: Fn(Params, Arc<Context>) -> Result<R, Error> + Clone + Send + Sync + 'static,
+		E: for<'a> Into<Error>,
```

Comment:
Unneeded `for<'a>`?

Reply:
Good find, I removed this one and the other one! It was a leftover from my experiments with directly using the `ErrorObject<'a>`